cclib/attributes/attribute.py

```diff
@@ -61,7 +61,7 @@ def typecheck(self):
             )
 
 
-class aonames(_Attribute):
+class Aonames(_Attribute):
```

Comment:
`AoNames`? `AONames`?

Reply:
I like that first letter leaves no real question and can see this being easy to remember as developing. Obviously AONames makes the most  sense in terms of being clear, so open to that too.